csharp/ql/src/utils/modelgenerator/internal/CaptureSummaryFlowQuery.qll

```diff
@@ -79,6 +77,22 @@ private class Activate extends ActiveConfiguration {
  * Captured Model:
  * ```Summaries;BasicFlow;false;AssignToArray;(System.Int32,System.Int32[]);Argument[0];Argument[1].Element;taint```
  */
+module ThroughFlow = TaintTracking::MakeWithState<ThroughFlowConfig>;
```

Comment:
You've moved the qldoc from the predicate below to this module - that looks accidental.

Reply:
Yes. This problem will disappear when moving the module instantiations and predicates.